encrypt-core/encrypt-core-common/src/main/java/org/apache/shardingsphere/encrypt/strategy/impl/AESEncryptor.java

```diff
@@ -43,44 +44,47 @@
 @Getter
 @Setter
 public final class AESEncryptor implements Encryptor {
-    
+
     private static final String AES_KEY = "aes.key.value";
-    
+
     private Properties properties = new Properties();
-    
+
     @Override
     public String getType() {
         return "AES";
     }
-    
+
     @Override
     public void init() {
     }
-    
+
     @Override
     @SneakyThrows
     public String encrypt(final Object plaintext) {
+        if (StringUtil.isNullOrEmpty(plaintext)) {
```

Comment:
The empty string can be encrypted, is it? Only null value can not encrypt.

Reply:
if value is null, the encryption method will get a 'null' string result.